pages/_app.js

```diff
@@ -21,12 +21,16 @@ export default class HomePage extends App {
         hiddenMenu.classList.add('HideMenu');
       }
 
-      if (mobileMenu.classList.contains('HideMenu') && showMobileMenu) {
-        mobileMenu.classList.add('ShowMenu');
-        mobileMenu.classList.remove('HideMenu');
-      } else {
-        mobileMenu.classList.add('HideMenu');
-        mobileMenu.classList.remove('ShowMenu');
+      if (hiddenMenu.classList.contains('MobileMenu') && showMobileMenu) {
```

Comment:
The problem with responsiveness is here, but we'll remove the menu function with JS soon.. So keep in this way